mrbelvedereci/plan/tasks.py

```diff
@@ -30,3 +33,28 @@ def run_scheduled_daily():
 @django_rq.job('short')
 def run_scheduled_hourly():
     return run_scheduled('hourly')
+
+@django_rq.job('short')
+def create_github_webhook(pk):
+    """Create a webhook if it doesn't exist yet."""
+    plan_repo = PlanRepository.objects.get(pk=pk)
+    if plan_repo.plan.type == 'manual':
```

Comment:
can this check happen before delaying the job? no need to get a worker all involved...

Reply:
Sure, good call.